File: Practice/src/coding_basic_training/ShuffleString.js
function solution(str1, str2) {
    var answer = '';
    for(let i=0;i<str1.length;i++){
        answer += str1.charAt(i) + str2.charAt(i);
    }
    return answer;
    // join 함수 이용
    // return [...str1].map((x, idx)=> x+str2[idx]).join("");
}

File: Practice/src/coding_basic_training/ShuffleString.py
def solution(str1, str2):
    answer = ''
    for i in range(0, len(str1)):
        answer += str1[i] + str2[i]
    # 리스트 컴프리헨션 버전
    # answer = ''.join([str1[i] + str2[i] for i in range(len(str1))])
    return answer

